backend/conduit/tags/views.py

```diff
@@ -58,9 +68,30 @@ def unfavorite_an_article(slug):
     return tag
 
 @blueprint.route('/api/tags/<slug>/members', methods=('GET',))
-@marshal_with(profile_schemas)
 def get_members_from_tag(slug):
```

Comment:
I think it would be better to create a schema in the serializers.py for tags called TagMembership where it returns only the members and moderators. That way you can reduce the number of code written